scripts/vep_backend/vep_batch_backend.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+"""Run VEP in parallel using batch backend"""
+
+import click
+from cpg_utils import to_path
+from cpg_utils.hail_batch import dataset_path, output_path
+from cpg_workflows.batch import get_batch
+from cpg_workflows.jobs.vep import add_vep_jobs
+
+
+@click.command()
+@click.option('--vep-version', help='Version of VEP', default='105')
+@click.option('--vcf-path', required=True, help='Path to VCF to run VEP on')
+@click.option(
+    '--output-ht',
+    required=True,
+    help='Path to where finished VEP-annotated VCF will be output',
+)
+@click.option(
+    '--scatter-count',
+    required=False,
+    help='Number of fragments to generate; default is 2',
+)
+def main(vep_version: str, vcf_path: str, output_ht: str, scatter_count: int):
+    """
+    Run VEP in parallel using Picard tools intervals as partitions.
+    """
+
+    b = get_batch(f'Run VEP with Batch Backend, VEP v{vep_version}')
+    add_vep_jobs(
+        b=b,
+        # I'm not sure what 'to_path' does here - is it needed?
+        vcf_path=to_path(dataset_path(vcf_path)),
+        # would something like this work to get the tmp path from the output_path input?
```

Comment:
in your example run command from the README I think this would deliver 

`gs://cpg-tx-adapt-test-tmp/vep_batch/v0/vcf_fragments`

Reply:
Yeah, I just don't want to add too many input parameters, so thought it'd be nice to 'elegantly' use something from the input we already have